### ISSUE TITLE
Mark as completed button doesn't work

### ISSUE THREAD

**Issue by picklu** [bnmnetp/runestone#616]
Mark as completed button doesn't work in the "Executing Python in this Book" chapter.

**Comment by bnmnetp** [comment]
Which course?   I have checked and it is working for me.  If you could let me know the course you are in, and your username I can try to do a bit more checking.

**Comment by picklu** [comment]
The book is "How to Think Like a Computer Scientist" and the page is for "Executing Python in this Book". I have checked several times but the results were the same. Another thing, everytime I log in with my user id "picklu" I am redirected to a page "http://interactivepython.org/runestone/static/IntroCompScience/index.html#" that says "Sorry, Something went wrong". Then if I click to link " Instructors Page " I can use the book. May be this is because I signed up as an instructor for this book.

**Comment by bnmnetp** [comment]
There is no content for the course named IntroCompScience.  It looks like the course was created incorrectly.  Did you make it?  If you just want to read the book then you should change your profile so you are registered for ``thinkcspy``  If you are an instructor and want to teach a class then you need to rebuild this book.

**Comment by picklu** [comment]
Thank you for the information. However, that problem is still there. In the subsection of "Executing Python in this Book"  of "How to think like a computer scientist", the button "Mark As Completed" does not work.

**Comment by bnmnetp** [comment]
OK, I still cannot reproduce the problem.   If you could check the javascript console in your browser and paste in any messages you see here that may help track it down.

Brad

**Comment by picklu** [comment]
When I pressed the button "Mark as completed" I got these errors on the consoles of Chrome and Firefox for this page "http://interactivepython.org/runestone/static/thinkcspy/GeneralIntro/SpecialWaystoExecutePythoninthisBook.html".
However, other pages also showed the same error message on Firefox.

Chrome:
Uncaught TypeError: Cannot read property 'tagName' of null
	isValidSelection @ selection.js:45
	check @ selection.js:74

FireFox:
Empty string passed to getElementById().
http://interactivepython.org/runestone/static/thinkcspy/_static/jquery.js
Line 3
